### PR TITLE
Material

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/themes/material/templates/body.array.cell.edit.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="editor">
+	<button type="text"
```
Comment (etokareva) [inline src/themes/material/templates/body.array.cell.edit.html:2]: button type="button"?

File: demo/index.js [hunk 2]
```diff
@@ -25,15 +25,21 @@ require('./index.scss');
...
+	.indexOf('?theme=default') >= 0
+	? themeDefault
+	: themeMaterial;
+
 const dependencies = [
 	ngRoute,
 	ngAnimate,
 	ngArea,
 	ngSanitize,
 	'ngMaterial', // WTF?
```
Comment (etokareva) [inline demo/index.js:40]: unnecessary comment?

File: src/themes/material/templates/body.array.cell.edit.html (hunk 1 above)
Comment (klumba12) [reply src/themes/material/templates/body.array.cell.edit.html:2]: it's not correct editor
just some html that is not used

File: demo/index.js (hunk 2 above)
Comment (klumba12) [reply demo/index.js:40]: no
its WTF)

**Review by etokareva — APPROVED** [review]

**Review by titovmx — APPROVED** [review]

**Pull request merged**